src/components/GeneralModal/GeneralModal.jsx

```diff
@@ -23,14 +31,24 @@ const GeneralModal = ({ openModal, setOpenModal, title, message }) => {
           <div className={classes.paper}>
             <h2>{title}</h2>
             <p>{message}</p>
-            <Button
-              className={classes.buttonGrad}
-              classvariant="contained"
-              type="button"
-              onClick={() => setOpenModal(false)}
-            >
-              cerrar
-            </Button>
+            <div className={classes.buttonsModal}>
+              <Button
+                className={classes.buttonGrad}
+                classvariant="contained"
+                type="button"
+                onClick={redirectToOrder}
```

Comment:
This route should be sent from father to son by props

Reply:
The route was not sent as a prop because the routing was handled by the father component and rendered as to children inside the modal